### ISSUE TITLE
ImproperlyConfigured: GEOIP_DATABASE setting is defined, but file does not exist.

### ISSUE THREAD

**Issue by lucas03** [Miserlou/django-easy-timezones#18]
I am having problem when pushing new app with django-easy-timezones. Basicly, I need to call `manage.py collectstatic` to put files GeoLiteCityv6.dat and GeoLiteCity.dat to static directory. But when I run `manage.py collectstatic`, following error is raised.  

    remote: django.core.exceptions.ImproperlyConfigured: GEOIP_DATABASE setting is defined, but file does not exist.
    remote: ln: creating symbolic link `/var/lib/openshift/user/app-root/runtime/repo//wsgi/static/media': No such file or directory
Neverending cycle. 

I also have a suggestion. Instead of a need to update geoip database manually, how about using https://github.com/futurecolors/django-geoip ?

**Comment by Miserlou** [comment]
I don't think you want the dat file in static, then it'll be publicly
available..

**Comment by lucas03** [comment]
Why not? That file is publicly available anyway.

**Comment by Miserlou** [comment]
..okay..

So, what is the problem?

d-e-t comes with the dat files included now, you shouldn't even need to
have them in your project.

**Comment by lucas03** [comment]
oh, I did not know that. Ok, thanks. Probably I wrote this issue to early before thinking about it enough.

**Comment by lucas03** [comment]
I am still missing those files:
   
    django.core.exceptions.ImproperlyConfigured: GEOIP_DATABASE setting is defined, but file does not exist.
I installed this app like this: `pip install django-easy-timezones==0.6.2`. Then I deleted `GEOIP_DATABASE` and `GEOIPV6_DATABASE` variables, because you said they are optional.
But I am still getting `ImproperlyConfigured` error.
I can see that GEOIP files are in your pip package - https://pypi.python.org/pypi/django-easy-timezones
but pip somehow does not use them? 
This is in my `dist-packages`(/usr/local/lib/python2.7/dist-packages) folder:
   
    $ ls | grep easy_time
    django_easy_timezones-0.6.2.dist-info
    easy_timezones

    $ ls easy_timezones/
    __init__.py  __init__.pyc  middleware.py  middleware.pyc  signals.py  signals.pyc  tests.py  tests.pyc  urls.py  urls.pyc  utils.py  utils.pyc  views.py  views.pyc

Should not be those GEOIP files in easy_timezones folder?

**Comment by Miserlou** [comment]
You're right, they are supposed to be. This seems to be a packaging problem, but I'm not sure why that's happening. For now, just set the paths explicitly and I'll fix this in a few h.

**Comment by a1Gupta** [comment]
Even I am getting same error.
Also I downloaded them from repo and added explicitly. Now I am getting Invalid database type, expected City error

**Comment by coler-j** [comment]
Is this issue fixed?

**Comment by thesunlover** [comment]
why don't you disable this GeoLiteCity check?

**Comment by NightHawk451** [comment]
I am also getting this error: GEOIP_DATABASE setting is defined, but file does not exist.

**Comment by bassamghouzi** [comment]
I am also getting this error

**Comment by pydevink** [comment]
This error is still present

**Comment by jjhelfman** [comment]
I am also getting this error caused by line 24 of middleware.py:     
```
    raise ImproperlyConfigured("GEOIP_DATABASE setting is defined, but file does not exist.")
django.core.exceptions.ImproperlyConfigured: GEOIP_DATABASE setting is defined, but file does not exist.
```
